purescript.html.markdown

```diff
@@ -2,194 +2,207 @@
 language: purescript
 contributors:
     - ["Fredrik Dyrkell", "http://www.lexicallyscoped.com"]
+    - ["Thimoteus", "https://github.com/Thimoteus"]
 ---
 
 PureScript is a small strongly, statically typed language compiling to Javascript.
 
 * Learn more at [http://www.purescript.org/](http://www.purescript.org/)
-* Documentation: [http://docs.purescript.org/en/latest/](http://docs.purescript.org/en/latest/)
+* Documentation: [http://pursuit.purescript.org/](http://pursuit.purescript.org/)
 * Book: Purescript by Example, [https://leanpub.com/purescript/](https://leanpub.com/purescript/)
 
-```haskell
+```purescript
 
 --
 -- 1. Primitive datatypes that corresponds to their Javascript
 -- equivalents at runtime.
 
+> import Prelude
 -- Numbers
-1 + 7*5 :: Number -- 36
+> 1.0 + 7.2*5.5 :: Number -- 40.6
```

Comment:
I'm not sure I see the point of all these `>`. They're just adding clutter, and making syntax highlighting worse...